ChangeLog

```diff
@@ -40,6 +40,10 @@ Release date: TBA
 
   Closes #6101
 
+* The main checker name is now ``main`` instead of ``master``.
```

Comment:
```suggestion
* The main checker name is now ``main`` instead of ``master``. The configuration does not need to be updated as sections' name are optional.
```